external-crates/move/crates/move-vm-runtime/src/jit/execution/translate.rs

```diff
@@ -103,6 +103,9 @@ impl PackageContext<'_> {
     }
 
     fn try_resolve_function(&self, vtable_entry: &VirtualTableKey) -> Option<VMPointer<Function>> {
+        if vtable_entry.package_key != self.runtime_id {
+            return None;
+        }
```

Comment:
I don't know why, but this lookup mechanism seems unsatisfactory. Maybe a doc comment would make me happy.